internal/app/knitgateway/gateway.go

```diff
@@ -112,60 +356,3 @@ func (c *certForwardingInterceptor) WrapStreamingHandler(handlerFunc connect.Str
 	// should never be called
 	return handlerFunc
 }
-
-func makeHTTPClient(config ServiceConfig) (connect.HTTPClient, error) {
```

Comment:
I moved this into `config.go` since we now create the client while building/validating config.